Source/Core/Cartesian3.js

```diff
@@ -1,11 +1,13 @@
 /*global define*/
 define([
+        './check',
```

Comment:
Can't more `DeveloperError`s in this file be change, e.g.,
```javascript
        if (!defined(left) || !defined(right)) {
            throw new DeveloperError('left and right are required.');
        }
```
Perhaps search for `DeveloperError` in this file and see which are reasonable to change.